plugins/drs/cluster/condensed/src/main/java/org/apache/cloudstack/cluster/Condensed.java

```diff
@@ -21,78 +21,61 @@
 
 import com.cloud.host.Host;
 import com.cloud.offering.ServiceOffering;
-import com.cloud.utils.Pair;
 import com.cloud.utils.Ternary;
 import com.cloud.utils.component.AdapterBase;
 import com.cloud.vm.VirtualMachine;
+import org.apache.cloudstack.framework.config.ConfigKey;
 
 import javax.naming.ConfigurationException;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 
 import static org.apache.cloudstack.cluster.ClusterDrsService.ClusterDrsImbalanceThreshold;
-import static org.apache.cloudstack.cluster.ClusterDrsService.ClusterDrsMetric;
+import static org.apache.cloudstack.cluster.ClusterDrsService.ClusterDrsMetricType;
 
 public class Condensed extends AdapterBase implements ClusterDrsAlgorithm {
```

Comment:
@vishesh92 you will need to extend Configurable interface and add "ClusterDrsImbalanceSkipThreshold" in getConfigKeys() method to save this new config in the database.